ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -49,6 +52,9 @@ export default class StatusDropdown extends React.Component {
         this.props.frameworks,
       ),
     };
+
+    this.isWeekend = false;
+    this.timeToTriage = 3;
```

Comment:
We could move this variable in `ui/perfherder/perf-helpers/constants.js`, makes more sense in this way.